.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+# Set update schedule for GitHub Actions
+
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
```

Comment:
I assume it should say daily (or the command change to weekly)

Reply:
It should be weekly, just corrected it (I thought for a quick test I try it with daily, but this seems to work only in the default branch)